cmd/use.go

```diff
@@ -16,6 +16,9 @@ import (
 // FileSelectedRefs stores selection, is copied from github.com/qri-io/qri/repo/fs/files.go
 const FileSelectedRefs = "/selected_refs.json"
 
+// QriRefFilename links the current working folder to a dataset by containing a ref to it.
+const QriRefFilename = ".qri-ref"
```

Comment:
ugh I hate to bikeshed, but I think this should be `.qri_ref` to keep consistent to our file naming conventions from `.qri` repo, which we inherited from go language conventions. fight me

Reply:
If we're going to change it, the rfc should be updated to match. It is currently using ".qri-ref", and I feel like the rfc should be the place to bikeshed about that. It is still in review.

Arguments in favor of ".qri-ref":
1) Though our source code is written in go, the external interface that users see on their filesystem is not, itself, the same as go. In other words, a separate reimplementation of our binary in another programming language would not follow go language conventions.
2) Following git for inspiration: in the .git repo, dash (-) is used for separating words, such as "packed-refs", "hooks/pre-commit", *unless* the words are entirely uppercase, like "ORIG_HEAD" and "COMMIT_EDITMSG".

Arguments in favor of ".qri_ref":
3) We should be consistent with the convention established in "~/.qri/". This alone seems reason enough to dismiss (1) and (2), if you want to go forward with this change.

Reply:
I should, not that it's very important, but my personal preference is dash (-) to separate words over (_), only because it doesn't require the shift key to type on my keyboard. It's what lisp uses and I've always liked it better. Fwiw.

Reply:
thx for comments on process, totally agreed we should debate this stuff over in RFC land. I'll move the discussion over there. Great points made here tho 😃